src/Tests/InstrEngineTests/NaglerInstrumentationMethod/InstrumentationEngineApi.cpp

```diff
@@ -33,7 +36,11 @@ const WCHAR InstrumentationEngineModule[] = _T("MicrosoftInstrumentationEngine_x
 #define MOD_TYPE HMODULE
 #define MOD_LOAD(_N) (GetModuleHandle(_N))
 #define SYM_LOAD(_M, _S) (GetProcAddress(_M, _S))
-
+#define GET_LAST_ERROR() \
```

Comment:
> GET_LAST_ERROR [](http://example.com/codeflow?start=7&length=15)

You could just as easily make this a function, and it would be easier to understand:

```
void AssertLastError()
{
#ifdef PLATFORM_UNIX
// linux/mac here
#else
// windows stuff
#endif
}
```

Reply:
remove the macro